### DISCUSSION TITLE
tflint does not recognise `removed` blocks

### DISCUSSION THREAD

**Discussion by jamiekt** [terraform-linters/tflint#1995] [Q&A]
### Summary

We run super-linter (ghcr.io/super-linter/super-linter:v5) to lint our terraform code. We turn on `tflint` by setting env var 
```yaml
jobs:
    run-lint:
        env:
            VALIDATE_TERRAFORM_TFLINT: true
```

Unfortunately its failing right now:

```
2024-02-27 15:43:52 [INFO]   Linting [TERRAFORM_TFLINT] files...
2024-02-27 15:43:52 [INFO]   ----------------------------------------------
2024-02-27 15:43:52 [INFO]   ----------------------------------------------
2024-02-27 15:43:52 [INFO]   ---------------------------
2024-02-27 15:43:52 [INFO]   File:[/github/workspace/path/to/file.tf]
╷
│ Error: Unsupported block type
│ 
│   on file.tf line 1:
│    1: removed {
│ 
│ Blocks of type "removed" are not expected here. Did you mean "moved"?
╵
```

`removed` blocks were added in terraform 1.7 (https://www.hashicorp.com/blog/terraform-1-7-adds-test-mocking-and-config-driven-remove). It would be great if `tflint` supported linting of files that contain `removed` blocks.

In the meantime, is it possible to ignore this error? We've tried:

```
# tflint-ignore: unsupported_block_type
```
but doing so is not having the desired effect

### Command

Unknown, its wrapped up in super-linter

### Terraform Configuration

```terraform
removed {
  from = snowflake_schema_grant.staging_ownership
  # The lifecycle block instructs Terraform not to destroy the underlying resource
  lifecycle {
    destroy = false
  }
}
```

### TFLint Configuration

```HCL
// https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md
config {
  module = false
  force = false
}

plugin "aws" {
  enabled = true
}

// disable pinned resource rule because we need to use master version of "github.com/org/tf-hf-rds"
rule "terraform_module_pinned_source" {
  enabled = false
}
```

### Output

```Shell
╷
│ Error: Unsupported block type
│ 
│   on file.tf line 16:
│   16: removed {
│ 
│ Blocks of type "removed" are not expected here. Did you mean "moved"?
╵
```

### TFLint Version

v0.48.0.0 (I think, that's what I'm assuming from this line: https://github.com/super-linter/super-linter/blob/a8150b40c89574adb5f68bf9502b890a236a06b3/Dockerfile#L15)

### Terraform Version

1.7

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows

**Comment by wata727** [comment]
This error is probably not due to TFLint, since it only validates the bare minimum schema. Please consider the possibility that other tools are giving the error.
If you still think that TFLint is the cause, please tell us the minimum steps to reproduce it without using super-linter.
